packages/react-server-gulp-module-tagger/README.md

```diff
@@ -0,0 +1,45 @@
+# react-server-gulp-module-tagger
+
+A [gulp](http://gulpjs.com) plugin for tagging [react-server](https://www.npmjs.com/package/react-server) logger instances with information about the module they're being used in.
+
+To transpile your React for use with [React Server](https://www.npmjs.com/package/react-server), install gulp and the plugin
+
+    npm i -D gulp react-server-gulp-module-tagger
+
+Then add the task to your gulpfile
+
+    const gulp = require('gulp');
+    const tagger = require('react-server-gulp-module-tagger');
+    gulp.task('compile', () => {
+        gulp.src('src')
+            .pipe(tagger())
+            .pipe(gulp.dest('dist'));
+    });
+
+Compile task might also use [Babel](https://babeljs.io) with the [React Server Babel preset](https://www.npmjs.com/package/babel-preset-react-server)
+
+    const gulp = require('gulp');
+    const babel = require('gulp-babel');
+    const tagger = require('react-server-gulp-module-tagger');
+
+    gulp.task('compile', () => {
+        gulp.src('src')
+            .pipe(tagger())
+            .pipe(babel({ presets: ['react-server'] }))
+            .pipe(gulp.dest('dist'));
+    });
+
+Given a [React Server](https://www.npmjs.com/package/react-server) getLogger call, adds the correct arguments to keep the server and the browser in sync.
+
+For example, given the following three files
+
+#### src/foo.js
+    const logger = require("react-server").logging.getLogger(__CACHE__);
+
+#### src/bar.js
+    let logger = require("react-server").logging.getLogger(__LOGGER__);
+
+#### src/baz.js
+    var logger = require("react-server").logging.getLogger(__CHANNEL__);
```

Comment:
`__CACHE__` and `__CHANNEL__` seem like funny examples to use here w/ a logger. They plop in the same info, but they're typically used in different contexts. (Which leads me to believe maybe our replacement string should be `__MODULE_CTX__` or something)